client/go/outline/config/module.go

```diff
@@ -100,6 +100,19 @@ func NewDefaultTransportProvider(tcpDialer transport.StreamDialer, udpDialer tra
 	// Stream dialers.
 	streamDialers.RegisterSubParser("shadowsocks", NewShadowsocksStreamDialerSubParser(streamEndpoints.Parse))
 
+	// IPTable support (TCP anly).
+	streamDialers.RegisterSubParser("ip-table", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
```

Comment:
Do we need the `-`? What is the naming convention here?
```suggestion
	streamDialers.RegisterSubParser("iptable", func(ctx context.Context, input map[string]any) (*Dialer[transport.StreamConn], error) {
```